tests/test_api_circuit_breaker.py

```diff
@@ -0,0 +1,101 @@
+import responses
+import pytest
+import re
+from freezegun import freeze_time
+from app import app
+from apistar.test import TestClient
+from idunn.blocks.wikipedia import WikipediaBreaker
+from time import sleep
+
+from .test_full import fake_all_blocks
+
+@pytest.fixture(scope="module", autouse=True)
+def breaker_test():
+    """
+    We define here settings specific to tests
+    for the circuit_breaker in order to avoid
+    any waste of time with real timeout and
+    failmax
+    """
+    WikipediaBreaker.init_breaker()
```

Comment:
useless method no ?

Reply:
not really: its purpose is to instantiate lazily the breaker to avoid circular dependencies by wrapping the 'import settings'.
Any better way to access the settings ?